aports/main/postmarketos-mkinitfs/mkinitfs.sh

```
@@ -111,9 +111,13 @@ get_binaries()
 	if [ "${deviceinfo_msm_refresher}" == "true" ]; then
 		BINARIES="${BINARIES} /usr/sbin/msm-fb-refresher"
 	fi
-	if [ -f /usr/sbin/fbdebug ]; then
-		BINARIES="${BINARIES} /usr/sbin/fbdebug"
-	fi
+	for file in "/etc/postmarketos-mkinitfs/files"/*.files; do
+		[ -f "$file" ] || continue
+		while IFS= read -r line; do
+			[ -f "$line" ] || continue
```

The hooks should always have all files mentioned in their lists, because they depend on the packages providing these files, right?
That means if a file mentioned in a line does not exist, this is a bug in the packaging or in the file list. So I would `exit 1` with an error message here instead of silently ignoring it.

---

I've tried like you suggested, to return an empty string and check the empty string in `copy_files`.
At the beginning it was solved, but a lot of problems started to raise when testing `pmbootstrap initfs hook_add debug-shell` with a non-existing file in the "list". 
Something related to the packages installed but the trigger failed and also the `.pmbootstrap` virtual package still present.
I've tried to add a try/catch in `pmb.chroot.apk.install` and call `apk del .pmbootstrap` in case of error but it keeps triggering the `postmarketos-mkinitfs`'s post-install script and raise some other errors.
Patch: https://pastebin.com/NeVcEvMF

---

Thanks for trying it out! Looks like I didn't think this through. I've pushed an alternative version, that puts the check into a new function (so we can simply `exit` from there) and runs it at the beginning. What do you think?